src/api/app/models/package.rb

```diff
@@ -42,6 +42,13 @@ class ReadSourceAccessError < APIException
   class IllegalFileName < APIException; setup 'invalid_file_name_error'; end
   class PutFileNoPermission < APIException; setup 403; end
 
+  unless defined?(BINARY_EXTENSIONS)
+    BINARY_EXTENSIONS = %w{.0 .bin .bin_mid .bz .bz2 .ccf .cert .chk .der .dll .exe .fw
```

Comment:
Have you tried if 
```BINARY_EXTENSIONS ||= ...```
would also solve the issue? Would be a lot nicer in my opinion :smile: